docs/concepts/gnovm.md

```diff
@@ -24,4 +24,59 @@ This enables non-web3 developers to contribute without requiring an understandin
 
-Check the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) to enhance GnoVM, Gno, and stdlibs.
+## Enhance or modify gnoVM
+
+To enhance or modify gnoVM, Gno, and stdlibs, familiarize yourself with the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile). 
+
+The following are parts of the gnoVM Makefile: environment variables, dev tools, test suite, and code gen. See the sections below for more details on each part.
+
+### Environment variables
+
+Environment variables define the virtual machine settings. Refer to the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) for reference. 
+
+These variables can be overwritten in the command line by passing the variable name and value. For example, to update the test suite timeout from the default `30m` to `15m`, pass `GOTEST_FLAGS= -v -p 1 -timeout=15m`. 
+
+Environment Variable | Definition | Default
+-------------------- | ---------- | ---------
+`CGO_ENABLED`        | Enable cgo to use any C code. Use may require additional dependencies, and is not strictly required by any tm2 code. See [Go's documentation on cgo with the go command](https://pkg.go.dev/cmd/cgo#hdr-Using_cgo_with_the_go_command). | cgo is disabled by default. `CGO_ENABLED` must be exported; use `export CGO_ENABLED`.
+`GOFMT_FLAGS`       | Change formatting of Go output. `-w` flag writes the output to the destination flag. See [Go's documentation on gofmt flags](https://pkg.go.dev/cmd/gofmt). | Uses the standard Go formatting by default.
+`GNOFMT_FLAGS`      | Change formatting of Gno output. `-w` flag writes the output to the destination flag. | Uses the standard Gno formatting by default.
+`GOIMPORTS_FLAGS`   | Adds Go formatting flags for `make imports`. | Uses values defined in `GOFMT_FLAGS` by default.
+`GOTEST_FLAGS`      | Modifies Go test suites through flags. | Prints verbose output (`-v`), defines the number of programs that can be run in parallel (`-p 1`), and times out after 30 minutes (`-timeout=30m`). 
+`GNOROOT_DIR`       | Sets the default GNOROOT. | In local development, sets GNOROOT to the supposed path of the Gno repository clone.
+`GOBUILD_FLAGS`     | Sets Go build flags in an argument list. See [Go's build flags documentation](https://pkg.go.dev/cmd/go). | Sets the arguments to pass on each Go tool link invocation to the Gno project.
+`GOTEST_COVER_PROFILE` | Defines the file where to place the cover profile. | The output location is set to `cmd-profile.out` by default/
+
+### Dev tools
+
+The dev tools section defines aliases for Go commands. See the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) for comprehensive Go commands corresponding to the aliases.
+
+Alias command | Definition
+------------- | ----------
+`build`       | Builds Gno with `GNOBUILD_FLAGS`.
```

Comment:
I think putting just `Gno` here is ambiguous -- this is a binary that we use to achieve certain functionality with the GnoVM